hw2/src/findPrimeFactor/FindPrimeFactor.java

```diff
@@ -31,6 +31,11 @@ public static int findPrimeFactor (List<Integer> a, List<Integer> b)
         // Also note that a[] and b[] need not be of the same length.
 
 
+        //check empty list
```

Comment:
I added additional tests where list a is empty and was thrown and `IndexOutOfBoundsException`. Checking for empty lists resolved it. Are you guys satisfied with this?